src/scripts/App.ts

```diff
@@ -11,8 +11,13 @@ class App {
         Preload.load(App.debug).then(function () {
             OakItemRunner.initialize();
             UndergroundItem.initialize();
-            App.game = new Game(new Breeding(), new Pokeballs(), new Wallet());
+            App.game = new Game(new Breeding(), new Pokeballs(), new Wallet(), new BadgeCase(16));
```

Comment:
This 16 is something of a magic number, and happens to be incorrect as we count each  elite4 and champion as badges (see the numbers in BadgeCase.Badge enum, and issue #476)

Reply:
Yeah excellent point. Later I was thinking of creating some sort of a `GameFactory` with methods like `createEasyPrestige()`, `createMediumPrestige()`, `createChallenge1()`, etc...

The reason for this parameter was so that the badge case doesn't display badges you can't get during this game, which I realize I still have to implement.

Maybe we should find a different way to store the elite four, adding them as badges is kind of iffy anyway.